src/tsolvers/stpsolver/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+target_sources(tsolvers
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPSolver.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPStore.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPMapper.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPEdgeGraph.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPGraphManager.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPModel.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/SafeInt.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/Converter.h"
+
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPSolver_implementations.hpp"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPStore_implementations.hpp"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPMapper_implementations.hpp"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPGraphManager_implementations.hpp"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/STPModel_implementations.hpp"
```

Comment:
Change STPSolver_implementations.hpp => STPSolver.cc etc.?

Reply:
The problem is that these files contain templated classes (templated over the Value type, which is different for RDL and for IDL) so they cannot be .cc files.

Reply:
Right, I remember having discussed this with you now!